libs/agora/gene-comparison-tool/src/lib/components/gene-comparison-tool-filter-list/gene-comparison-tool-filter-list.component.html

```diff
@@ -20,17 +20,17 @@
           [isVisible]="significanceThresholdActive"
           (clearEvent)="removeSignificanceThresholdFilter()"
         ></agora-gene-comparison-tool-filter-list-item>
-        <ng-container *ngFor="let filter of filters; index as i">
-          <ng-container *ngFor="let option of filter.options; index as j">
+        @for (filter of filters; track filter; let i = $index) {
+          @for (option of filter.options; track option; let j = $index) {
```

Comment:
One of Nx migration steps is responsible of this conversion. Note that we have an ESLint rule that show a warning when we use the old syntax. I have seen some warnings after running the migration, which indicate that there may still be code that use the old syntax. This can be address in follow-up PRs.